secboot/secboot_tpm.go

```diff
@@ -44,17 +46,17 @@ const (
 )
 
 var (
-	sbConnectToDefaultTPM                  = sb.ConnectToDefaultTPM
-	sbMeasureSnapSystemEpochToTPM          = sb.MeasureSnapSystemEpochToTPM
-	sbMeasureSnapModelToTPM                = sb.MeasureSnapModelToTPM
-	sbBlockPCRProtectionPolicies           = sb.BlockPCRProtectionPolicies
-	sbActivateVolumeWithTPMSealedKey       = sb.ActivateVolumeWithTPMSealedKey
-	sbAddEFISecureBootPolicyProfile        = sb.AddEFISecureBootPolicyProfile
-	sbAddEFIBootManagerProfile             = sb.AddEFIBootManagerProfile
-	sbAddSystemdEFIStubProfile             = sb.AddSystemdEFIStubProfile
-	sbAddSnapModelProfile                  = sb.AddSnapModelProfile
-	sbSealKeyToTPMMultiple                 = sb.SealKeyToTPMMultiple
-	sbUpdateKeyPCRProtectionPolicyMultiple = sb.UpdateKeyPCRProtectionPolicyMultiple
+	sbConnectToDefaultTPM                  = sb_tpm2.ConnectToDefaultTPM
+	sbMeasureSnapSystemEpochToTPM          = sb_tpm2.MeasureSnapSystemEpochToTPM
+	sbMeasureSnapModelToTPM                = sb_tpm2.MeasureSnapModelToTPM
+	sbBlockPCRProtectionPolicies           = sb_tpm2.BlockPCRProtectionPolicies
+	sbActivateVolumeWithTPMSealedKey       = sb_tpm2.ActivateVolumeWithSealedKey
+	sbefiAddSecureBootPolicyProfile        = sb_efi.AddSecureBootPolicyProfile
+	sbefiAddBootManagerProfile             = sb_efi.AddBootManagerProfile
+	sbefiAddSystemdStubProfile             = sb_efi.AddSystemdStubProfile
+	sbAddSnapModelProfile                  = sb_tpm2.AddSnapModelProfile
+	sbSealKeyToTPMMultiple                 = sb_tpm2.SealKeyToTPMMultiple
+	sbUpdateKeyPCRProtectionPolicyMultiple = sb_tpm2.UpdateKeyPCRProtectionPolicyMultiple
```

Comment:
@anonymouse64 when @chrisccoulson did the port from sb.FOO to sb_efi.FOO he renamed the constants from sbFOO to sbefiFoo.

I'm now wondering if my sb.FOO to sb_tpm2.FOO port should follow suite and rename sbFOO things to sbtpm2FOO.

Is there any guidance around this? ie. should I rename sbConnectToDefaultTPM  to sbtpm2ConnectToDefaultTPM and so on?

Reply:
the pattern you have here is plenty readable and fine to me, perhaps @pedronis has further opinions but I think this is fine